lightning-c-bindings/demo.cpp

```diff
@@ -331,9 +331,12 @@ class PeersConnection {
 		t2 = std::thread(&sock_read_data_thread, pipefds_1_to_2[0], &sock2, &net2);
 
 		// Note that we have to bind the result to a C++ class to make sure it gets free'd
-		LDK::CResult_CVec_u8ZPeerHandleErrorZ con_res = PeerManager_new_outbound_connection(&net1, ChannelManager_get_our_node_id(&cm2), sock1);
+		LDKCOption_NetAddressZ remote_network_address = {
+			.tag = LDKCOption_NetAddressZ_None,
```

Comment:
nit: calling `COption_NetAddressZ_none()` is a lot cleaner, and maybe can even be inlined in the method call.